citi-onboarding/celeiro-dos-bambas
# Feature/impact fix

## What I did

- I reduced the margins of the Impact section, removed the fixed position of the footer and added a margin between the contact and footer sections.

## How to test

1. In terminal:
   1. `git checkout main`
   1. `git pull`
   1. `git checkout feature/impactFix`
   1. `git pull`
   1. `yarn run develop` in `server/` folder
   1. `yarn run start` in `client/` folder
 2. Go to [http://localhost:3000/](http://localhost:3000/)
 3. Check if the adjustments in the footer, contact and impact sections are as expected.
